src/bookmark/bookmark.service.ts

```diff
@@ -54,4 +59,27 @@ export class BookmarkService {
       throw new BadRequestException();
     }
   }
+
+  async deleteBookmark(
+    user: User,
+    bookmarkId: string,
+    groupId: string,
+  ): Promise<void> {
```

Comment:
Do you think we should declare return type even when it's automatically inferred?

Reply:
When I removed return type, I got this error in eslint.

> Missing return type on function.eslint[@typescript-eslint/explicit-module-boundary-types](https://typescript-eslint.io/rules/explicit-module-boundary-types)